setup.cfg

```diff
@@ -50,13 +51,12 @@ test =
     pytest-asyncio; python_version>="3.4"
     graphene>=2.0,<3
     gevent
-    graphene>=2.0
     graphene_django
     mock; python_version<"3"
     django==1.11.*; python_version<"3"
     channels==1.*; python_version<"3"
-    django==2.*; python_version>="3"
-    channels==2.*; python_version>="3"
+    django==3.*; python_version>="3"
+    channels==3.*; python_version>="3"
```

Comment:
Is it safe to assume all python >3 users will use django ?

Reply:
Not sure what you mean?

Reply:
I meant "will use django 3"